examples/README.md

```diff
@@ -1991,6 +1991,30 @@ To avoid these issues, the `ghpc_stage` function can be used to copy a file (or
 The `ghpc_stage` function will always look first in the path specified in the blueprint. If the file is not found at this path then `ghpc_stage` will look for the staged file in the deployment folder, if a deployment folder exists.
 This means that you can redeploy a blueprint (`gcluster deploy <blueprint> -w`) so long as you have the deployment folder from the original deployment, even if locally referenced files are not available.
 
+### Reservation for Scheduled Maintenance
+
+A maintenance event is when a compute engine stops a VM to perform a hardware or
+software update which is determined by the host maintenance policy. This can
+also affect the running jobs if the maintenance kicks in. Now, Customers can control scheduled maintenance
```

Comment:
We CAN NOT "control scheduled maintenance", it will happen regardless of toolkit settings. Let's phrase it in terms of protection against "sudden death of jobs" )